R/LinkGSF.R

```diff
@@ -182,3 +182,34 @@ link_gsf_dsld <- function(
         contribution_fname = "link_dsld_contribution"
     )
 }
+
+
+# link_gsf_identity-class ----
+
+#' `link_gsf_identity`
+#'
+#' This class extends the general [`link_gsf_abstract`] for the identity of the
```

Comment:
Similar comment here on the markdown syntax
```suggestion
#' This class extends the general [link_gsf_abstract()] for the identity of the
```